### PR TITLE
Update conditions.rst

### PR DESCRIPTION
**Pull request by kockodev**
Undocumented functionality -> injecting container parameters into Route condition

### CODE REVIEW

File: routing/conditions.rst [hunk 1]
```diff
@@ -87,3 +87,7 @@ variables that are passed into the expression:
 
     Because of this, using the ``condition`` key causes no extra overhead
     beyond the time it takes for the underlying PHP to execute.
+
+.. parameters-block:: Injecting parameters
+
+    You can inject container parameters (e.g. `%kernel.environment%`) into condition, such as `@Route(condition="'%kernel.environment' in ['dev']")` or any other you want.
```
Comment (OskarStark) [inline routing/conditions.rst]:
```diff
- @Route(condition="'%kernel.environment' in ['dev']")
+ @Route(condition="'%kernel.environment%' in ['dev']")
````

???

Comment (rapemer) [reply routing/conditions.rst]: @OskarStark you have eagle eyes :+1:

File: routing/conditions.rst [hunk 2]
```diff
@@ -87,3 +87,10 @@ variables that are passed into the expression:
 
     Because of this, using the ``condition`` key causes no extra overhead
     beyond the time it takes for the underlying PHP to execute.
+
+.. _parameters:
```
Comment (OskarStark) [inline routing/conditions.rst]: do we need this?

Comment (rapemer) [reply routing/conditions.rst]: I don't know how it works, maybe not

Comment (OskarStark) [reply routing/conditions.rst]: I think you can remove it

File: routing/conditions.rst (hunk 1 above)
Comment (OskarStark) [reply routing/conditions.rst]: 🤣

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Lukáš! Before merging, we reworded it to add this note as commented code. It's something we do lately in docs to make them more concise. Thanks!

**Comment by OskarStark** [comment]
Does „matches“ work with arrays?

**Comment by javiereguiluz** [comment]
@OskarStark we reworded the example to not use that.

**Pull request merged**